src/pulse3D/metrics.py

```diff
@@ -171,8 +171,8 @@ def calculate_amplitudes(
             c10y = twitch_data.loc["force", "contraction", 10]
             r90x = twitch_data.loc["time", "relaxation", 90]
             r90y = twitch_data.loc["force", "relaxation", 90]
-            slope = (r90y - c10y) / (c10x - r90x)
-            twitch_base_y = c10y + slope * (twitch_peak_x - c10x)
```

Comment:
This doesn't need to change

Reply:
Sorry I removed it for testing and forget to add it back.